easydigitaldownloads/easy-digital-downloads
# Do not replace the cart contents if the ajax response is empty

Fixes #8719

Proposed Changes:
1. Update the JS to only replace the cart contents if the ajax response is not empty.

I did not update the corresponding ajax function as this is working, but I can.

To Test:
1. Enable two gateways (eg Test and Stripe).
2. Enable Wallet.
3. Enable taxes.
3. Make a deposit to the wallet.
4. On the checkout screen, switch to the second gateway.

In `master`, the cart will disappear. In this branch, it should not.
